systemd/README.md

```diff
@@ -0,0 +1,40 @@
+# Systemd Units
+
+This directory contains a systemd unit definitions. It runs a the sshnpd component
+of the ssh! no ports in a GNU screen, which can be attached to (for logging etc.)
+with `screen -r`
+
+## Installation
+
+The `sshnpd.service` files should be placed in `/etc/systemd/system` (as root).
+
+Modify the `sshnpd.service` unit to use the appropriate sender and target

```

Comment:
May also need to modify the user if not running on a pi

Reply:
agreed